systems/robotInterfaces/@Biped/planZMPTraj.m

```diff
@@ -171,3 +172,34 @@
   support_options.contact_groups = initial_support_groups;
   supp = RigidBodySupportState(biped, initial_supports, support_options);
 end
+
+function [rstep, lstep] = getSafeInitialSupports(biped, kinsol, steps, options)
+  % Make sure that our center of mass is within our initial available contact points by modifying the contact groups on the first two steps if necessary. 
+  if nargin < 4
+    options = struct('support_shrink_factor', 0.8);
+  end
+  com = getCOM(biped, kinsol);
+  all_supp_pts_in_world = zeros(3, 0);
+  for f = {'left', 'right'}
+    foot = f{1};
+    supp_groups = steps.(foot).walking_params.support_contact_groups;
+    supp_pts = biped.getTerrainContactPoints(biped.foot_body_id.(foot), {supp_groups});
+    supp_pts = supp_pts.pts;
+    supp_pts = bsxfun(@plus, mean(supp_pts, 2), options.support_shrink_factor * bsxfun(@minus, supp_pts, mean(supp_pts, 2)));
+    T_sole_to_world = poseQuat2tform(steps.(foot).pos);
+    T_sole_to_foot = biped.getFrame(steps.(foot).frame_id).T;
+    T_foot_to_world = T_sole_to_world / T_sole_to_foot;
+    supp_pts_in_world = T_foot_to_world * [supp_pts; ones(1, size(supp_pts, 2))];
```

Comment:
oh, whoops, yeah. I'd copied this logic from another function which did not have access to a kinsol. Fixed now.